presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoInstaller.java

```diff
@@ -74,6 +77,7 @@ private static void installUsingRpm(String pathToRpm, boolean checkDependencies)
         if (installRpm != 0) {
             throw new PrestoManagerException("Failed to install presto", installRpm);
         }
+        addTpchCatalog();
```

Comment:
The user can delete the catalog if it is not required. But I think tpch is a basic catalog which is needed.